code/server/models/User.js

```diff
@@ -0,0 +1,138 @@
+var Sequelize = require('sequelize');
```

Comment:
This file should not be here any more, when I added the data access layer I renamed this model to UserDB.